build_tools/scripts/utils.py

```diff
@@ -24,13 +24,7 @@ def check_and_get_output_lines(command: Sequence[str],
   print(f'Running: `{" ".join(command)}`')
   if dry_run:
     return None, None
-  process = subprocess.run(
-      command,
-      # TODO(#4131) python>=3.7: Use capture_output=True.
-      stderr=subprocess.PIPE,
-      stdout=subprocess.PIPE,
-      # TODO(#4131) python>=3.7: Replace 'universal_newlines' with 'text'.
-      universal_newlines=True)
+  process = subprocess.run(command, capture_output=True, text=True)
```

Comment:
I think this whole bit from this line to the end could be substituted with

```python
return subprocess.run(command, stdout=subprocess.PIPE, text=true, check=True).stdout.splitlines()
```

with no need to explicitly log stderr (and log it to stdout for some reason). The default behavior is to just use the existing stderr. I'm pretty sure that I wrote this at some point while fighting to understand `subprocess.run`, so mea culpa on this one.

@stellaraccident because I see you did something similar below with `_write_binary_stderr`, am I missing something?

Reply:
Done!